### PR TITLE
chore: create helper classes

### PR DESCRIPTION
**Pull request by y-lakhdar**
Just move some of the code into helper classes to prevent code duplication later on.

### CODE REVIEW

**Review by olamothe — APPROVED** [review]

File: src/help/axiosUtils.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export const axiosRequestHeaders = (apikey: string): AxiosRequestConfig => {
+  const authorizationHeader = {
+    Authorization: `Bearer ${apikey}`,
+  };
+
+  const documentsRequestHeaders = {
+    ...authorizationHeader,
+    'Content-Type': 'application/json',
+    Accept: 'application/json',
+  };
```
Comment (louis-bompart) [inline src/help/axiosUtils.ts:12]: Ain't the double object + spread a bit overkill?

Comment (louis-bompart) [inline src/help/axiosUtils.ts:12]: Ain't the double object + spread a bit overkill?

**Review by louis-bompart — APPROVED** [review]
lgtm, some unit tests could be nice.

File: src/help/axiosUtils.ts (hunk 1 above)
Comment (y-lakhdar) [reply src/help/axiosUtils.ts:12]: Yes, but it's because it will be reused multiple times later by both the stream and push strategies

**Pull request merged**